1-js/02-first-steps/04-variables/article.md

```diff
@@ -1,343 +1,344 @@
 
-alert(message); // Hello!
+alert(message); // Bună!
 ```
 
-We can also declare multiple variables in one line:
+De asemenea putem declara multiple variabile într-o singură linie:
 
 ```js no-beautify
-let user = 'John', age = 25, message = 'Hello';
+let user = 'John', age = 25, message = 'Bună';
 ```
 
-That might seem shorter, but we don't recommend it. For the sake of better readability, please use a single line per variable.
+Asta ar putea să apară mai scurt, dar nu recomandăm. De dragul unei lizibilități mai bune, te rugăm să folosești o singură linie de variabilă.
 
-The multiline variant is a bit longer, but easier to read:
+Varianta pe mai multe linii este puțin mai lungă, dar mai ușor de citit:
 
 ```js
 let user = 'John';
 let age = 25;
-let message = 'Hello';
+let message = 'Bună';
 ```
 
-Some people also define multiple variables in this multiline style:
+Unele persoane definesc, de asemenea, multiple variabile în acest stil pe mai multe linii:
 ```js no-beautify
 let user = 'John',
   age = 25,
-  message = 'Hello';
+  message = 'Bună';
 ```
 
-...Or even in the "comma-first" style:
+...Sau chiar în "prima-virgulă" ca stil:
 
 ```js no-beautify
 let user = 'John'
   , age = 25
-  , message = 'Hello';
+  , message = 'Bună';
 ```
 
-Technically, all these variants do the same thing. So, it's a matter of personal taste and aesthetics.
+Technic, toate aceste variante fac același lucru. Deci, este o problemă de preferință personală și de estetică.
 
 ````smart header="`var` instead of `let`"
-In older scripts, you may also find another keyword: `var` instead of `let`:
+În transcrieri mai vechi, ai putea găsi de asemenea un alt termen: `var` în loc de `let`:
 
 ```js
-*!*var*/!* message = 'Hello';
+*!*var*/!* message = 'Bună';
 ```
 
-The `var` keyword is *almost* the same as `let`. It also declares a variable, but in a slightly different, "old-school" way.
+Termenul `var` este *aproape* la fel ca și `let`. Declară de asemenea o variabilă, but într-un mod ușor diferit, de "școală-veche" .
 
-There are subtle differences between `let` and `var`, but they do not matter for us yet. We'll cover them in detail in the chapter <info:var>.
+Există diferențe subtile între `let` și `var`, dar nu contează pentru noi acum. O să le acoperim în detaliu în capitolul <info:var>.
 ````
 
-## A real-life analogy
+## O analogie din viața reală
 
-We can easily grasp the concept of a "variable" if we imagine it as a "box" for data, with a uniquely-named sticker on it.
+Putem înțelege ușor conceptul de "variabilă" dacă ne-o imaginăm drept o "cutie" pentru date, cu un abțibild cu nume unic pe ea.
 
-For instance, the variable `message` can be imagined as a box labeled `"message"` with the value `"Hello!"` in it:
+De exemplu, variabila `message` poate fi imaginată drept o cutie cu eticheta `"mesaj"` cu valoarea `"Bună!"` în ea:
 
 ![](variable.svg)
 
-We can put any value in the box.
+Putem pune orice valoare în cutie.
+
```

Comment:
aici sau pe undeva pe aici au iesit liniile din ordine. Recomand să păstram ordinea liniilor la fel, asta face integrarea viitoarelor update-uri mult mai usuara si lucrul in echipa mult mai efectiv.

Reply:
@bogdanbacosca am verificat toate liniile in article.md dupa Github. Acum sunt la rand dar vad ca tot nu apare ca resolved :(

Reply:
@Alexandra2888 O să verific și o să marchez eu. 🙂

Reply:
@bogdanbacosca Multumesc :)

Reply:
@Alexandra2888 dacă totul e ok și ai făcut toate modificările poți să scrii /done din nou pentru a cere alt review.

Reply:
/done

Reply:
@Alexandra2888 în căsuța principală din PR, nu aici. Robotul ascultă comentariile de acolo.